.golangci.yml

```diff
@@ -0,0 +1,124 @@
+issues:
```

Comment:
Adding this one here to remove the detailed linter. This would be added later by Terraform anyways.